src/app/lib/transaction.ts

```diff
@@ -79,6 +82,73 @@ export class OasisTransaction {
     return tw
   }
 
+  public static async buildStakingAllowTransfer(
+    nic: OasisClient,
+    signer: Signer,
+    to: string,
+    amount: bigint,
+  ): Promise<TW<oasis.types.StakingAllow>> {
+    const tw = oasis.staking.allowWrapper()
+    const nonce = await OasisTransaction.getNonce(nic, signer)
+    const beneficiary = await addressToPublicKey(to)
+
+    tw.setNonce(nonce)
+    tw.setFeeAmount(oasis.quantity.fromBigInt(0n))
+    tw.setBody({
+      beneficiary,
+      negative: false,
+      amount_change: oasis.quantity.fromBigInt(amount),
+    })
+
+    const gas = await tw.estimateGas(nic, signer.public())
+    tw.setFeeGas(gas)
+
+    return tw
+  }
+
+  public static async buildParatimeTransfer(
+    nic: OasisClient,
+    signer: Signer,
+    targetAddress: string,
+    fromAddress: string,
+    amount: string,
```

Comment:
This was updated to 
```ts
  public static async buildParatimeTransfer(
    nic: OasisClient,
    signer: Signer,
    transaction: ParaTimeTransaction, // for amount other methods use bigint
    fromAddress: string,
    runtimeId: string,
    runtimeDecimals: number,
  )
```

I think it should be more detailed:
```ts
  /**
   * Note: uses raw user input amounts in ROSE as parameter instead of 
   * normalized bigint base units, because it needs to convert to base units 
   * dependent on paratime decimals.
   */
  public static async buildParatimeTransfer(
    nic: OasisClient,
    signer: Signer,
    transaction: ParaTimeTransaction,
    fromAddress: string,
    runtimeId: string,
    runtimeDecimals: number,
  )
```